<a href="https://colab.research.google.com/github/Fish-Dory/HW-2/blob/main/Untitled0.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
expected = [50,50,50,50,50]
observed = [19, 50, 47, 53, 60]
import scipy.stats as stats

#perform Chi-Square Goodness of Fit Test
stats.chisquare(f_obs=observed, f_exp=expected)

Power_divergenceResult(statistic=4.36, pvalue=0.3594720674366305)

In [ ]:
import numpy as np

n = 10000
time_between_calls = np.random.default_rng().exponential(scale=10, size=n)

x = ((time_between_calls < 5).sum())/n
y = ((time_between_calls < 4).sum())/n
x-y

0.0625

Вероятность того, что клиент позвонит в ближайшие 4-5 минут составляет 0.0625